src/OpenSearch.Client/CommonAbstractions/Static/Infer.cs

```diff
@@ -31,11 +31,11 @@
 using System.Linq.Expressions;
 using System.Reflection;
 
-namespace Osc
+namespace OpenSearch.Client
 {
 	public static class Infer
 	{
-		public static readonly Indices AllIndices = Osc.Indices.All;
+		public static readonly Indices AllIndices = Client.Indices.All;
```

Comment:
This compiles, and passes right?

Reply:
Yup. You can see compilation is passing in the PR checks bellow.
<img width="847" alt="image" src="https://user-images.githubusercontent.com/99687200/171753670-77b55c06-6b9e-4c6b-a95c-26960e150351.png">

Not sure why `Client` is necessary there since `Infer` is in the same namespaces as `Indices` -- `OpenSearch.Client`. 🤷 

Vast majority of these changes were done by IntelliJ's Rename command. It missed a few spots, but generally worked.